src/Snakemake/rules/QC/samtools.smk

```diff
@@ -0,0 +1,13 @@
+rule samtools_stats:
+    input:
+        bam="DNA_bam/{sample}-ready.bam",
+    output:
+        "qc/{sample}/{sample}.samtools-stats.txt",
+    params:
+        extra="-t " + config["bed"]["bedfile"],
+    log:
+        "logs/qc/samtools_stats/{sample}.log",
+    singularity:
+        config["singularity"]["samtools"]
```

Comment:
change to config["singularity"].get("samtools", config["singularity"].get("default", ""))?

Reply:
done